### DISCUSSION TITLE
005

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#3] [General]
# 005

《大橘和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

https://rrorangeandfriends.site/posts/2024/005

**Comment by Zoromax555** [comment]
可以发一期python的知识库网站吗？求求了

**Reply by zhiyu1998** [reply]
我开源过一个，其他的话`006期`见 ツ

地址：https://zhiyu1998.github.io/Python-Basis-Notes/

**Reply by zhiyu1998** [reply]
第六期没有找到合适的，你可以持续关注下，后续我找到会放到周刊中
